packages/devextreme/js/localization/messages/en.json

```
@@ -155,6 +155,7 @@
         "dxDataGrid-columnFixingUnfix": "Unfix",
         "dxDataGrid-columnFixingLeftPosition": "To the left",
         "dxDataGrid-columnFixingRightPosition": "To the right",
+        "dxDataGrid-columnFixingStickyPosition": "To the sticky",
```

```suggestion
        "dxDataGrid-columnFixingStickyPosition": "Stick in place",
```

Please redo all localization files.